release notes/v1.2.0.md

```diff
@@ -0,0 +1,349 @@
+k6 v1.2.0 is here 🎉! This release includes:
+
+- Automatic extension resolution(previously Binary Provisioning) enabled for everyone
+- gRPC gets better handling of `NaN` and `Infinity` float values and easier health check.
+- browser module gets `page.route`, all the `page.getBy*` APIs, `locator.all()`, and `page.waitForURL`
+- a number of bugfixes and other small improvements
+
+## Breaking changes
+
+As per our [stability guarantees](https://grafana.com/docs/k6/latest/reference/versioning-and-stability-guarantees/),
+breaking changes across minor releases are allowed only for experimental features.
+
+### Breaking changes for experimental modules
+
+- The experimental Open Telemetry and Prometheus outputs now default to TLSv1.3. This should've been the default to begin with. It is not expected that anyone should be affected, apart from making it more secure for the metrics output to send messages.
+
+### Extensions support
+
+We have made multiple enhancements for running scripts with extensions:
+
+- [#5019](https://github.com/grafana/k6/pull/5019) Enables the feature by default. Also renames it as Automatic Extension Resolution and the environment variable to enables/disables it as `K6_AUTO_EXTENSION_RESOLUTION`.
+- [#4811](https://github.com/grafana/k6/pull/4811) Removes limitations for processing test scripts from stdin or using file extensions other than `.js` or `.tar`.
+- [#4812](https://github.com/grafana/k6/pull/4812) When running a test using a custom `k6` binary, the automatic extension resolution checks if the it includes all required extensions before attempting to provision a custom binary.
+' [#4921](https://github.com/grafana/k6/pull/4921), [#5013](https://github.com/grafana/k6/pull/5013) Removes the requirement for GCK6 authentication for using the automatic extension resolution, and enables the feature for local execution (i.e., the `k6 run` command).
+- [#4922](https://github.com/grafana/k6/pull/4922) Supports running tests using community extensions.
+- [#5039](https://github.com/grafana/k6/pull/5039) Disable automatic resolution when start custom binaries, including for old versions of k6.
+
+Previously, k6 only supported official extensions. As of the PR [#4922](https://github.com/grafana/k6/pull/4922), it's now possible to use any extension listed in the [extensions registry](https://grafana.com/docs/k6/latest/extensions/explore/) by setting the `K6_ENABLE_COMMUNITY_EXTENSIONS` environment variable.
+
+```
+K6_ENABLE_COMMUNITY_EXTENSIONS=true k6 run script.js
+```
+
+Note: Community extensions are supported only for local executions (`k6 run` or `k6 cloud run --local-execution`). Tests executed in GCK6 still are limited to use the official extensions.
+
+### Handling of NaN and Infinity float values in gRPC [#4631](https://github.com/grafana/k6/pull/4631)
```

Comment:
Do we need a ## New features section just above this? It feels like the breaking changes are only to do with the extension support.